apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -169,7 +169,7 @@ const SlotPicker = ({
     timeZone,
   });
 
-  const slots = useMemo(() => ({ ..._1, ..._2 }), [_1, _2]);
+  const slots = useMemo(() => ({ ..._2, ..._1 }), [_1, _2]);
```

Comment:
@emrysal change the order here so merging the single day availability into the monthly one.